meson.build

```diff
@@ -973,7 +973,7 @@ if features['d3d11']
 endif
 
 wayland = {
-    'deps': [dependency('wayland-client', version: '>= 1.15.0', required: get_option('wayland')),
+    'deps': [dependency('wayland-client', version: '>= 1.22.0', required: get_option('wayland')),
```

Comment:
Instead of bumping the required version, could you make a `HAVE_WAYLAND_1_22` define like how there already is an existing `HAVE_WAYLAND_1_20` in wayland_common.c and then guard the functions?

Reply:
How would I got about doing this? I don't see where `HAVE_WAYLAND_1_20` is defined.

Reply:
It's right up there at the top.
https://github.com/mpv-player/mpv/blob/6d422b3edc00a36dde5e1b073a07938d98720f3a/video/out/wayland_common.c#L54-L56

Reply:
Eh, sorry, I missed it when grepping, just assumed that `#define` wouldn't be in the same file… Anyways, done.